### DISCUSSION TITLE
Jailmaker for a CUPS printing server

### DISCUSSION THREAD

**Discussion by Kosztyk** [Jip-Hop/jailmaker#193] [Q&A]
Hi,

I am trying to install a cups printing server on a jail to use is with an older printer Lexmark E230 but unfortunately Cups server do not see it.
If i execute lsusb it list it, can be that teh usb devices needs to be passtrought jail? If yes, can someone help me with some ideas?

Thank you

<img width="822" alt="Screenshot 2024-06-01 at 15 56 25" src="https://github.com/Jip-Hop/jailmaker/assets/36381705/6b7887a4-f760-43c4-bc1a-ee5d414d38e6">
<img width="630" alt="Screenshot 2024-06-01 at 15 57 43" src="https://github.com/Jip-Hop/jailmaker/assets/36381705/5581955d-61c9-4246-8f44-03d7485f4720">

**Comment by Jip-Hop** [comment]
Yeah maybe device passthrough is required. What is the output of `ls -la /dev` on the host (not in the jail)? You can passthrough a device with `--bind=/dev/nameofdevice`.
